scipy/ndimage/filters.py

```diff
@@ -337,6 +337,14 @@ def prewitt(input, axis=-1, output=None, mode="reflect", cval=0.0):
     %(output)s
     %(mode)s
     %(cval)s
+
+    Examples
+    --------
+    >>> import scipy.misc
+    >>> ascent = scipy.misc.ascent()
+    >>> from scipy.ndimage import prewitt
+    >>> result = prewitt(ascent)
+    >>> scipy.misc.imshow(result)
```

Comment:
at the top of the example: ``import matplotlib.pyplot as plt``

and then here: ``plt.imshow(result)``